Kickstarter-iOS/Views/Controllers/SettingsAccountViewController.swift

```diff
@@ -9,7 +9,9 @@ final class SettingsAccountViewController: UIViewController {
   @IBOutlet private weak var tableView: UITableView!
 
   private let dataSource = SettingsAccountDataSource()
-  fileprivate let viewModel: SettingsAccountViewModelType = SettingsAccountViewModel()
+  fileprivate let viewModel: SettingsAccountViewModelType = SettingsAccountViewModel(
+    SettingsAccountViewController.viewController(for:)
```

Comment:
In order to avoid needing access to `UIViewController` types in the view model I'm passing this function in here which was previously contained inside the view model. I think we could potentially improve the way this logic is handled but just did this for now to get it building with minimal changes. Let's discuss 👍